submissions/eoneoff/weather-api/getWeather.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const axios = require('axios');
+
+const weather = axios.create({
+  baseURL: 'http://api.openweathermap.org/data/2.5/',
+  params: {
+    APPID: 'c4ad7974f3977f8f388a60b5c0267caa'
+  }
+});
+
+module.exports.getWeather = async function (mode, location, units) {
+  const data = (await weather.get(mode, {
+    params: {
+      q: location,
+      units: units
+    }
+  })).data;
+  const wData = [];
+  if (mode === 'weather') {
+    const dData = makeWeatherData(data);
+    dData.city = data.name;
+    wData.push(dData);

```

Comment:
I think it would be better to create a "processingData" function and just call it in both situations